source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/GenerateFormContainer.tsx

```diff
@@ -14,16 +22,143 @@ export interface IGenerateFormContainerProps {
 const GenerateFormContainer: React.FunctionComponent<
   React.PropsWithChildren<IGenerateFormContainerProps>
 > = ({ acquisitionFileId, View }) => {
-  const generateLetter = useGenerateLetter();
+  const [fullRecipientsList, setFullRecipientsList] = useState<LetterRecipientModel[]>([]);
+  const { getPersonConcept, getOrganization } = useApiContacts();
```

Comment:
In my current PR I added a "GetOrganizationConcept" that gets the Api_Organization concept (instead of the IEditableOrganization that getOrganization returns) 

getOrganization is old code from when each form had unique models. Now we have moved towards sharing "concept" models.

Ideally we should be able to sync my changes into your PR

Reply:
Updated with GetOrganizationConcept method.